docs/user-manual/en/configuration-index.md

```diff
@@ -118,6 +118,11 @@ system-property-prefix | Prefix for replacing configuration settings using Bean
 [network-check-list](network-isolation.md) | The list of pings to be used on ping or InetAddress.isReacheable
 [network-check-ping-command](network-isolation.md) | The command used to oping IPV4 addresses
 [network-check-ping6-command](network-isolation.md) | The command used to oping IPV6 addresses
+[critical-analyzer](critical-analysis.md) | Enable or disable the critical analysis (default true)
```

Comment:
I'm not sure what you mean here. I'm adding some doc changes. Can you change it after merged?